far/fileedit.hpp

```diff
@@ -120,6 +120,7 @@ class FileEditor final: public window,public EditorContainer
 	int GetId() const { return m_editor->EditorID; }
 	FileEditor* GetById(int ID) { return GetId()==ID?this:nullptr; }
 	void AutoDeleteColors() const { m_editor->AutoDeleteColors(); }
+	void UpdateKeyBar() { InitKeyBar(); }
```

Comment:
As Sonar says, this shadows `window::UpdateKeyBar`.
Please rename it or just make `InitKeyBar` public.